packages/devtools/lib/src/memory/memory.dart

```diff
@@ -144,6 +206,22 @@ class MemoryScreen extends Screen {
     }
   }
 
+  // TODO(terry): Need to record the GC event in the timeline.
+  Future<Null> _gcNow() async {
+    gcNowButton.disabled = true;
+
+    try {
+      // 'reset': true to reset the object allocation accumulators
+      await serviceManager.service.callMethod('_getAllocationProfile',
+          isolateId: _isolateId, args: {'gc': 'full'});
+    } catch (e) {
+      // TODO(terry): Need something probably logging?
```

Comment:
is this a message we want to display to users. There is an API in this code to display a toast for users with an error.
If not, the TODO and print is fine.

Reply:
I think just a toast that the GC couldn't be done is enough.  I'll print out the error detail to the console.

Reply:
we have support to display toasts. I would go ahead and display one.